### DISCUSSION TITLE
How to set TIME field?

### DISCUSSION THREAD

**Discussion by fugacityengr** [tortoise/tortoise-orm#976] [Q&A] [answered]
I specifically want a column for just time alone (no date and no time zone) such as "08:00:00".
PostgreSQL offers the TIME field for that purpose, however I do not know how to specify it when declaring the columns.
How does one accomplish that?

**Comment by long2ice** [comment, marked as answer]
See https://tortoise.github.io/fields.html#extending-a-field, or welcome to make a pr in https://github.com/tortoise/tortoise-orm/blob/develop/tortoise/contrib/postgres/fields.py
